services/movie_session.py

```diff
@@ -0,0 +1,34 @@
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time, movie_id, cinema_hall_id):
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date=None):
+    if session_date is not None:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id):
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(session_id, show_time=None,
+                         movie_id=None, cinema_hall_id=None):
+    if show_time is not None:
+        MovieSession.objects.filter(id=session_id).update(show_time=show_time)
+    if movie_id is not None:
+        MovieSession.objects.filter(id=session_id).update(movie=movie_id)
+    if cinema_hall_id is not None:
+        MovieSession.objects.filter(id=session_id)\
+            .update(cinema_hall=cinema_hall_id)
```

Comment:
You don't need to `MovieSession.objects.filter` three times in this conditional block, you can do this before this block.